designer/server/src/main/scala/pl/touk/nussknacker/ui/factory/NussknackerAppFactory.scala

```diff
@@ -70,6 +73,27 @@ class NussknackerAppFactory(processingTypeDataStateFactory: ProcessingTypeDataSt
     Resource.pure[IO, MetricRegistry](new MetricRegistry)
   }
 
+  private def createFEStatisticsRepository(
+      clock: Clock,
+      config: QuestDbConfig,
+      system: ActorSystem
+  ): Resource[IO, FEStatisticsRepository[Future]] = {
+    val repository = if (config.enabled) {
+      QuestDbFEStatisticsRepository.create(system, clock, config)
+    } else {
+      logger.info("QuestDb is disabled - collecting FE statistics is skipped")
```

Comment:
do you think the `info` level is ok here? It's not so important information for the user.